src/SDK/Language/Node.php

```diff
@@ -12,31 +12,84 @@ public function getName(): string
         return 'NodeJS';
     }
 
-    /**
-     * @param array $parameter
-     * @param array $spec
-     * @return string
-     */
-    public function getTypeName(array $parameter, array $spec = []): string
+    public function getTypeName(array $parameter, array $method = []): string
     {
         if (isset($parameter['enumName'])) {
             return \ucfirst($parameter['enumName']);
         }
         if (!empty($parameter['enumValues'])) {
             return \ucfirst($parameter['name']);
         }
-        return match ($parameter['type']) {
-            self::TYPE_INTEGER,
-            self::TYPE_NUMBER => 'number',
-            self::TYPE_STRING => 'string',
-            self::TYPE_FILE => 'InputFile',
-            self::TYPE_BOOLEAN => 'boolean',
-            self::TYPE_OBJECT => 'object',
-            self::TYPE_ARRAY => (!empty(($parameter['array'] ?? [])['type']) && !\is_array($parameter['array']['type']))
-                ? $this->getTypeName($parameter['array']) . '[]'
-                : 'string[]',
-            default => $parameter['type'],
-        };
+        switch ($parameter['type']) {
+            case self::TYPE_INTEGER:
+            case self::TYPE_NUMBER:
+                return 'number';
+            case self::TYPE_ARRAY:
+                if (!empty(($parameter['array'] ?? [])['type']) && !\is_array($parameter['array']['type'])) {
+                    return $this->getTypeName($parameter['array']) . '[]';
+                }
+                return 'string[]';
+            case self::TYPE_FILE:
+                return "File";
```

Comment:
I've switched this to `File` on purpose. The InputFile class now returns an instance of the File class.

File is a web Javascript standard API[1] and is available on most edge runtimes including vercel[2].
It's available in Node.js from version 18, and we use `node-fetch-native` to fallback on a polyfill where it doesn't exist.

[1] https://developer.mozilla.org/en-US/docs/Web/API/File
[2] https://nextjs.org/docs/pages/api-reference/edge
[3] https://nodejs.org/docs/latest/api/buffer.html#class-file

Reply:
Does this change the service method parameter types? We can use `File` internally, but should still expect an `InputFile` in the user-facing methods like `storage.createFile`

Reply:
The `InputFile` helpers import will break on edge runtimes (due to its dependency on Node's `fs` module), so if we want `storage.createFile` to work on edge runtimes, we'll have to use the builtin `File` object.

If we use `InputFile` in the service method parameters then it is more obvious to the user that the helpers exist, but it will only work in Node-like environments